Client/WebFilters/AdBlockEngineManager.swift

```diff
@@ -395,10 +392,12 @@ public actor AdBlockEngineManager: Sendable {
           return
         }
         
-        if engine.deserialize(data: data) {
-          continuation.resume()
-        } else {
-          continuation.resume(throwing: CompileError.couldNotDeserializeDATFile)
+        DispatchQueue.main.async {
```

Comment:
Shouldn't we use the serial queue on Adblock stats?

Reply:
Needs further investigation, for the time being it's safer to use main queue here

Reply:
Do not remove this line or we will have the threading pause/delay again.